src/snowcli/cli/snowpark/commands.py

```diff
@@ -348,6 +348,7 @@ def _deploy_single_object(
             "external_access_integrations"
         ),
         "secrets": object_definition.get("secrets"),
+        "imports": object_definition.get("imports", []),
```

Comment:
Are imports added to schema definition?

Reply:
Good point. Added.
BTW. aren`t you supposed to rest, decorate the tree and prepare bigos?